XSAccessor.xs

```diff
@@ -369,8 +369,8 @@ STMT_START {                                                                 \
 STMT_START {                                                                 \
   autoxs_hashkey *hk_ptr = get_hashkey(aTHX_ obj_hash_key, obj_hash_key_len);\
   INSTALL_NEW_CV_WITH_PTR(name, xsub, hk_ptr);                               \
-  hk_ptr->key = (char*)cxa_malloc(obj_hash_key_len+1);                       \
-  cxa_memcpy(hk_ptr->key, obj_hash_key, obj_hash_key_len);                   \
+  cxa_malloc(hk_ptr->key, obj_hash_key_len+1, char);                         \
```

Comment:
Trailing space.

Reply:
`obj_hash_key_len+1` -> `obj_hash_key_len + 1`. (Yes, some of these are inconsistent in the original.)